src/generic/server/streaming.rs

```diff
@@ -56,7 +57,15 @@ where T: Future<Item = Response<S>,
         let response = response.into_http();
 
         // Map the response body
-        let (head, body) = response.into_parts();
+        let (mut head, body) = response.into_parts();
+
+        // Set the content type
+        // TODO: Don't hard code this here
+        let content_type = "application/grpc+proto";
```

Comment:
If the `CONTENT_TYPE` associated const moved to the `Encoder` trait, I think we could say
```rust
head.headers.insert(
    header::CONTENT_TYPE,
    header::HeaderValue::from_static(content_type),
);
```
instead...